packages/broker/test/integration/per-node-metrics.test.ts

```diff
@@ -81,21 +115,13 @@ describe('per-node metrics', () => {
         })
 
         storageNode = await startBroker({
-            name: 'storageNode',
+            name: 'storageNode', 
             privateKey: storageNodeAccount.privateKey,
+            networkPort: networkPort,
```

Comment:
AFAIK `networkPort` is no longer used, where did this come from?

Reply:
without it the storage node doesn't start and the tests crash on the `before` sections

Reply:
Actually Eric was right, without the network port it does work fine, removed it